my-app/src/Card.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './App.css';
+
+function Card(props){
+    const [userProfilePicture, setProfilePicture] = React.useState('Loading...')
```

Comment:
Totally your call, but I'd probably be more inclined to use one piece of state here called `user` that is an object with all of the properties that we need. Some people prefer to keep their state more simple/broken out, so it's really just personal preference, but I find it a little easier to read. Also, I'd probably just use a `null` value as the default state, and then conditionally render a loading message if the user isn't present. This can get a little weird with stuff like the `img` src, which will just look like a broken link.  Something like:

```jsx
const [user, setUser] = useState(null);

// Later in the promise callback

setUser({
  firstName: res.data.first_name,
  lastName: res.data.last_name,
  profilePicture: res.data.avatar,
  email: res.data.email
})

// Finally, when rendering 

{ !user ? <p>Loading...</p> : // render the user info }
```